plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2588,6 +2590,18 @@ elementgui.tool.stays_in_grid_when_crafting=Does item stay in crafting grid when
 elementgui.tool.damaged_on_crafting=<html>Damage item instead on crafting<br><small>Make sure to enable "stay in crafting grid" and that item is damageable
 elementgui.tool.tooltip_tip=<html>Special information about the tool:<br><small>Use comma to separate entries
 elementgui.tool.needs_a_name=Tool needs a name
+elementgui.villager_profession.display_name=Profession display name:
+elementgui.villager_profession.point_of_interest=Point of interest
```

Comment:
Since we don't really use the POI term in UI, I'd say this should be changed e.g. to:
```properties
elementgui.villager_profession.profession_block_validator=Profession block
```
and also moved below the next line.

Reply:
Done!